client/blindbox/command/azure-sev/attestation/main.go

```diff
@@ -0,0 +1,257 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+package main
+
+import (
+	"encoding/base64"
+	"encoding/json"
+	"flag"
+	"fmt"
+	"net/http"
+	"os"
+	"strconv"
+	"strings"
+
+	"ACI_Attest/attest"
```

Comment:
Ouf of curiosity, is that the code Microsoft uses to generate the attestation without any change?

Reply:
It is, with a lot of extra stuff removed.